docs-src/content/using-cubic-sdk/client-configs.md

```diff
@@ -0,0 +1,108 @@
+| enable_confusion_network | No | `false` | Toggles the creation of a confusion network, consisting of multiple alternatives transcriptions.  Cubic models must support confusion networks for this field to be populated. |
+| audio_channels | No | `[0]` (mono) | Specifies which channels of a multi-channel audio file to be transcribed, each as their own individual audio stream. |
+
+## Use cases
+
+The most basic use case is getting a formatted transcript.  This would simply need a config such as:
+
+```json
+{
+    "model_id": "1",
+    "audio_encoding": "raw"
+}
+```
+
+and your resulting transcript would be found at `results.alternatives[*].transcript`.
+
+> Note: Prior to `cubicsvr` v2.9.0 and SDK-Cubic v1.3.0, the field `results.alternatives[*].transcript` was populated with either the raw or the formatted transcription depending on the `enable_raw_transcript` config.  After these changes, raw transcripts have been pushed to a new field `results.alternatives[*].raw_transcript`, and only populated when `enable_raw_transcript` is set to true.
+
+------
+
+Sometimes, only the Raw transcript is desired.  In this case, there are two options.  
+
+1. Disable server-side formatting and use the above config.  Retrieve the raw transcript from the `results.alternatives[*].transcript` field.
+2. Specify the `enable_raw_transcript = true` flag, and access the field `results.alternatives[*].raw_transcript`.
+
+> Note: Prior to `cubicsvr` v2.9.0 and SDK-Cubic v1.3.0, the field `results.alternatives[*].transcript` was populated with either the raw or the formatted transcription depending on the `enable_raw_transcript` config.  After these changes, raw transcripts have been pushed to a new field `results.alternatives[*].raw_transcript`, and only populated when `enable_raw_transcript` is set to true.
+
+------
+
+Another use case would be getting both the formatted and raw transcript.  This can be done using this config.
+
+```json
+{
+    "model_id": "1",
+    "audio_encoding": "raw",
+    "enable_raw_transcript": "true"
+}
+```
+
+The formatted transcript would be found at `results.alternatives[*].transcript`, and the raw transcript would be found at `results.alternatives[*].raw_transcript`.
+
+------
+
+If you needed to dive deeper in word level timestamps, say for subtitles, then you can use this config to enable those word-level timestamps.  Please note the inclusion of `enable_raw_transcript`.
+
+```json
+{
+    "model_id": "1",
+    "audio_encoding": "raw",
+    "enable_raw_transcript": true,
+    "enable_word_time_offsets": true
+}
+```
+
+Timestamps will be found at `results.alternatives[*].words[*].start_time` and `results.alternatives[*].words[*].duration`.
+
+------
+
+If you needed to dive deeper in word level confidences, i.e. for displaying a lighter color for less confident words, then you can use this config to enable those word-level confidence values.  Please note the inclusion of `enable_raw_transcript`.
```

Comment:
This line sounds too copy-and-pasted from the description above for enabling word-level timestamps.